### DISCUSSION TITLE
Airflow webserver is not opening

### DISCUSSION THREAD

**Discussion by akshaya091299** [apache/airflow#35940] [General]
### Apache Airflow version

2.7.3

### What happened

I have install the apcahe airflow in ubuntu server and after successful installation, tried with airflow webserver -p 8080 command however, Web server is showing "Oops something went wrong"..
Please help here

### How to reproduce

step 1: install the ubuntu server.
step 2:

sudo apt-get install software-properties-common 
sudo apt-add-repository universe 
sudo apt-get update 
sudo apt-get install python-pip

export SLUGIFY_USES_TEXT _UNIDECODE=yes 
sudo pip install apache-airflow

above steps followed and after installation, airflow webserver -p 8080

### Operating System

windows 10

### Versions of Apache Airflow Providers

apache-airflow                           2.7.3
apache-airflow-providers-common-sql      1.8.0
apache-airflow-providers-ftp             3.6.1
apache-airflow-providers-http            4.7.0
apache-airflow-providers-imap            3.4.0
apache-airflow-providers-sqlite          3.5.0

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by potiuk** [comment]
converted to discussion. Please do not open issues when you want help to troubleshoot your installation. In the message you mentioned - there is a sugestion you shoudl look at logs, and you should listen to the suggestion - without the logs you will never find what's wrong. logs will likely help you to diagnose your issue. And the more information you provide here, the bigger chance is someone will actually decide to spend their free time on trying to help you to find a solution.
